### PR TITLE
Backend Typescript Conversion

### PR DESCRIPTION
**Pull request by janto393**
I converted the existing JS backend server to a Typescript format.

PSA: server.js -> src/index.tsx

### CODE REVIEW

**Comment by janto393** [comment]
@jamilgonzalez can you look over the TS configuration files to see if I missed anything important. To my knowledge, it should be almost identical to front-end except for the directories

File: server.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// if (process.env.NODE_ENV === 'production') 
+// {
+//   // Set static folder
+//   app.use(express.static(path.join(__dirname, 'front-end', 'build')));
+
+//   app.get('*', (request, res) => 
+// 	{
+//     res.sendFile(path.join(__dirname, 'front-end', 'build', 'index.html'));
+// 	});
+// }
```
Comment (jamilgonzalez) [inline server.ts]: Was this supposed to stay commented?

File: src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import * as dotenv from 'dotenv';
+
+const app = express();
+const PORT = process.env.PORT || 5000;
+
+app.get('/', (_, res) => {
+	res.status(200).send();
+});
```
Comment (jamilgonzalez) [inline src/index.tsx:9]: Getting a 'Cannot Get /' when I run this branch locally. Were you able to get a success while testing? I'm testing in browser.

**Review by jamilgonzalez — APPROVED** [review]
Looks good to me! Just wanna make sure the SQL code was supposed to be commented for this PR.

File: server.ts (hunk 1 above)
Comment (janto393) [reply server.ts]: The file was deleted on the latest commit since the code is now obsolete

**Pull request merged**